### DISCUSSION TITLE
use extract-text-webpack-plugin config languages css error

### DISCUSSION THREAD

**Discussion by daydayuphuan** [microsoft/monaco-editor#3655] [Q&A]
when i config option languangs 

![image](https://user-images.githubusercontent.com/7309887/42159131-3f927bf2-7e25-11e8-897b-19ae64a47079.png)

An exception error occurred in the compiled CSS

![image](https://user-images.githubusercontent.com/7309887/42159283-bfbf3360-7e25-11e8-9274-f5388f72009a.png)

**Comment by alexdima** [comment]
I think we somehow end up believing we are writing to a `.js` file. @daydayuphuan What is your complete webpack config (around CSS especially), as this appears to be some sort of bad interaction...

**Comment by daydayuphuan** [comment]
```
const extract = new ExtractTextPlugin({
    filename: (getPath) => {
        return getPath('style/[name].css').replace('js/', '');
    },
    allChunks: true
});
```

```
{
    test: /\.css$/,
    loader: extract.extract({
        fallback: 'style-loader',
        use: ['css-loader', 'postcss-loader']
    })
},
{
    test: /\.less$/,
    loader: extract.extract(['postcss-loader', 'less-loader'])
}
```
webpack version: 4.12.1
node version: v10.4.1
